### DISCUSSION TITLE
Cannot download releases https://support.hdfgroup.org/ftp/HDF5/releases via AppVeyor

### DISCUSSION THREAD

**Discussion by tbeu** [HDFGroup/hdf5#4805] [Breaking Change]
Removing https://support.hdfgroup.org/ftp/HDF5/releases (espcially https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-1.8.13/bin/windows/hdf5-1.8.13-win32-VS2010-static.zip or https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-1.8.13/bin/windows/hdf5-1.8.13-win32-VS2010-shared.zip) [breaks](https://ci.appveyor.com/project/tbeu/matio/builds/50538159/job/tsfhws338r2r10bu) my workflows. Where can I find old releases?

**Comment by lkurz** [comment]
Url is missing an index.html

https://support.hdfgroup.org/ftp/HDF5/releases/index.html

Urls in the ci look good.  I can download with browser.

Your CI shows error  "Could not create SSL/TLS secure channel."


________________________________
From: tbeu ***@***.***>
Sent: Wednesday, September 4, 2024 11:57:34 PM
To: HDFGroup/hdf5 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [HDFGroup/hdf5] https://support.hdfgroup.org/ftp/HDF5/releases is gone (Discussion #4805)


Removing https://support.hdfgroup.org/ftp/HDF5/releases (espcially https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-1.8.13/bin/windows/hdf5-1.8.13-win32-VS2010-static.zip or https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.8/hdf5-1.8.13/bin/windows/hdf5-1.8.13-win32-VS2010-shared.zip) breaks<https://ci.appveyor.com/project/tbeu/matio/builds/50538159/job/tsfhws338r2r10bu> my workflows. Where can I find old releases?

**Reply by tbeu** [reply]
Thanks. Correct and reported at https://github.com/appveyor/ci/issues/3930.
